package.json

```diff
@@ -0,0 +1,80 @@
+{
+  "name": "dataverse-frontend",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
+    "@types/node": "^16.18.12",
```

Comment:
Does this mean we should be using node 16.18.12? If so, I can't install it:

```
$ nvm install 16.18.12
Version '16.18.12' not found - try `nvm ls-remote` to browse available versions.
```

I tried with just `nvm install 16.18` and that worked, giving me node 16.18.1.

I see that in GitHub Actions there's a newer version of node. I'm making an assumption that I should use the version here in package.json but I'm not sure. 😅

Reply:
Ups, I forgot to add the node version to the package.json.

`@types/node` is the library for the TypeScript node types, so the version do not correspond to the node version.

I am currently using node v19 that is why in the Github Actions it appears as node 19, but node v16 would work as well.

I edited package.json to add the node and npm version

```
"engines" : {
    "npm" : ">=8.0.0",
    "node" : ">=16.0.0"
  },
```

The version used should be greater or equal than those but I'm using node v19 and npm v9. Maybe I should add it to the README too